src/doc/rustc/src/platform-support.md

```diff
@@ -34,6 +34,7 @@ Specifically they will each satisfy the following requirements:
 
 target | std | host | notes
 -------|-----|------|-------
+`aarch64-unknown-linux-gnu` | ✓ | ✓ | ARM64 Linux (kernel 4.2, glibc 2.17+) [^missing-stack-probes]
```

Comment:
Is mdbook support this link type? Because it seems that github markdown renderer doesn't.

Reply:
Yep.

![2020-10-22--14-24-01](https://user-images.githubusercontent.com/2299951/96871215-40262100-1472-11eb-9a2d-bfe3a95b34fd.png)